### ISSUE TITLE
‘Theme’ label is not associated with radio buttons

### ISSUE THREAD

**Issue by MicrosoftIssueBot** [microsoft/calculator#1178]
**Repro Steps:** 
1. Launch "Calculator" app and turn on narrator. 
2. Home screen will be displayed.
3. Navigate to menu button and activate it.
4. Navigate to "Graphing" list item and activate it.
5. Navigate to "Grid options" button and activate it.
6. Graph options pop up will open.
7. Navigate to 'theme' radio buttons.
8. Observe the issue.

**Actual Result:**
‘Theme’ label is not associated with radio buttons.

**Expected Result:**
‘Theme’ label should be associated with radio buttons.

**Comment by MicrosoftIssueBot** [comment]
This is your friendly Microsoft Issue Bot. I created this issue automatically as requested by a team member.
